code/mob/living/life/blood.dm

```diff
@@ -22,7 +22,8 @@
 		// 		src.blood_volume -= 1 * mult
 
 		var/mult = get_multiplier()
-
+		if(owner && HAS_MOB_PROPERTY(owner, PROP_TEMP_CHEM_EFFECTS))
+			mult *= get_temperature_mult()
```

Comment:
?? this is blood code, not chems

Reply:
I should probably name that something better. Or move that function onto the spleen. So, if they're warmer, they should bleed a lot more, but also stop bleeding a lot quicker too.